galgebra/mv.py

```diff
@@ -1306,13 +1306,22 @@ def simplify(self, modes=simplify):
             obj += coef * base
         return Mv(obj, ga=self.Ga)
 
+    def subs(self,subs_in,subs_out):
+        if isinstance(subs_in,(list,tuple)) and isinstance(subs_out,(list,tuple)):
+            for (sub_in,sub_out) in zip(subs_in,subs_out):
+                self.obj = self.obj.subs(sub_in,sub_out)
+        else:
+            self.obj = self.obj.subs(subs_in,subs_out)
+        return self
+
+    """
     def subs(self, d):
         # For each scalar coef of the multivector apply substitution argument d
         obj = sum((
             coef.subs(d) * base for coef, base in metric.linear_expand_terms(self.obj)
         ), S(0))
         return Mv(obj, ga=self.Ga)
-
+    """
```

Comment:
Covered by https://github.com/pygae/galgebra/pull/243 I think.

This implementation is wrong, and mutates `self.obj`, something which is a bad idea.